doc/Analytics.xml

```diff
@@ -3209,11 +3209,14 @@ Added Recognition rule engines</revremark>
           the current view port resulting from pan, title and zoom values so that these can be used
           independently from the view port. Of course rules with graphical elements located fully outside
           of the current view port will not be able to trigger any events.</para>
-        <para>As alternative the rules may include an element item called "PTZStatus" that acts as a
-          filter for activating the rule only when the view port matches the given values.The
+        <para>To realize this, the rules may include an element item called "PTZVector" that acts as a
+          filter for activating the rule only when the view port matches the given values. The
           ElementItem identifies the position of the device for which the rule has been setup. The
           corresponding ElementItemDescription resembles the following:</para>
-        <programlisting><![CDATA[<tt:ElementItemDescription Name="PTZStatus" Type="tt:PTZStatus">]]></programlisting>
+        <programlisting><![CDATA[<tt:ElementItemDescription Name="PTZVector" Type="tt:PTZVector">]]></programlisting>
+          As alternative the rules may include a "PresetToken" that acts in the same way as the "PTZVector". The
+          corresponding ElementItemDescription resembles the following:</para>
```

Comment:
Presettoken is simpleitemdescription, but in above statement its mentioned as elementitemdescription. This needs correction.

Reply:
Done